Multisig/UI/UI Library/Address/AddressCell.swift

```diff
@@ -9,8 +9,16 @@
 import SwiftUI
 
 struct AddressCell: View {
+
+    enum Style {
```

Comment:
instead of this, we already have the `AddressStyle` enum, maybe let's use that one?